### PR TITLE
i18n(es): Updated `contentful.mdx` & `recipes.mdx`

### PR DESCRIPTION
**Pull request by dreyfus92**
#### What kind of changes does this PR include?

- New or updated content
- Translated content

#### Description

- What does this PR change? Give us a brief description.
Updated `contentful.mdx` & `recipes.mdx` based on these commits f31000472e8fd9edf17473b01f65bd6730fdc837 & f2e2f9b99e7fb023d52be3848538e847378b3dfb

### CODE REVIEW

File: src/content/docs/es/guides/cms/contentful.mdx [hunk 1]
```diff
@@ -421,7 +426,7 @@ const { content, title, date } = Astro.props;
 </html>
 ```
 
-Navega a http://localhost:3000/posts/astro-is-amazing/ para asegurarte de que tu ruta dinámica esté funcionando.
+Navega a http://localhost:3000/ y has clic en uno de tus artículos para asegurarte de que tu ruta dinámica esté funcionando.
```
Comment (Waxer59) [inline src/content/docs/es/guides/cms/contentful.mdx:429]:
```suggestion
Navega a http://localhost:3000/ y haz clic en uno de tus artículos para asegurarte de que tu ruta dinámica esté funcionando.
```

**Review by Waxer59 — COMMENTED** [review]
Buena PR 🤠, te dejo una sugerencia

**Comment by dreyfus92** [comment]
Gracias por la revisión @Waxer59

**Review by Waxer59 — APPROVED** [review]
LGTM ✔

**Review by Yan-Thomas — APPROVED** [review]
LGTM! Thanks peeps 🚀

**Pull request merged**